applications/crossbar/src/modules/provisioner_util.erl

```diff
@@ -131,6 +131,15 @@ ensure_mac_cleansed(JObj) ->
             kzd_devices:set_mac_address(JObj, cleanse_mac_address(MacAddress))
     end.
 
+-spec get_new_device(cb_context:context()) -> kzd_devices:doc().
+get_new_device(Context) ->
+    JObj = cb_context:doc(Context),
```

Comment:
Could `JObj` be populated with default values from JSON schema validation that might be necessary later?

Reply:
@jamesaimonetti not at this time, no